File: src/dotnet/ArdsTools/Ards.NetCore.Entity/Models/Oblast.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Ards.NetCore.Entity.Models
{
    [Table("oblast", Schema = "adif")]
    public partial class Oblast
    {
        [Column("id")]
        public int Id { get; set; }
        [Required]
        [Column("code")]
        [StringLength(10)]
        public string Code { get; set; }
        [Required]
        [Column("name")]
        [StringLength(120)]
        public string Name { get; set; }
        [Column("is_deleted")]
        public bool IsDeleted { get; set; }
        [Required]
        [Column("comment")]
        [StringLength(120)]
        public string Comment { get; set; }
    }
}

File: src/java/ArdsToolAdifModels/src/com/ards/jpa/model/Oblast.java
package com.ards.jpa.model;

import java.io.Serializable;
import javax.persistence.*;


/**
 * The persistent class for the oblast database table.
 * 
 */
@Entity
@Table(name="oblast")
@NamedQuery(name="Oblast.findAll", query="SELECT o FROM Oblast o")
public class Oblast implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@GeneratedValue(strategy=GenerationType.AUTO)
	@Column(unique=true, nullable=false)
	private Integer id;

	@Column(nullable=false, length=10)
	private String code;

	@Column(nullable=false, length=120)
	private String comment;

	@Column(name="is_deleted", nullable=false)
	private Boolean isDeleted;

	@Column(nullable=false, length=120)
	private String name;

	public Oblast() {
	}

	public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getCode() {
		return this.code;
	}

	public void setCode(String code) {
		this.code = code;
	}

	public String getComment() {
		return this.comment;
	}

	public void setComment(String comment) {
		this.comment = comment;
	}

	public Boolean getIsDeleted() {
		return this.isDeleted;
	}

	public void setIsDeleted(Boolean isDeleted) {
		this.isDeleted = isDeleted;
	}

	public String getName() {
		return this.name;
	}

	public void setName(String name) {
		this.name = name;
	}

}